e2e-tests/cypress/integration/appellant-submission-check-your-answers.feature

```diff
@@ -0,0 +1,13 @@
+@wip
+Feature: A user checks their answers and wants to submit their appeal
+
+  Scenario: The user has valid data and wants to subnit their appeal
+    Given the user "has" provided enough data to submit an appeal
+    When the user confirms that they are happy with their answers
+    Then the user is asked to agree to the Terms and Conditions of the service
+
+
+  Scenario: The user has invalid data and wants to subnit their appeal
+    Given the user "has not" provided enough data to submit an appeal
+    When the user checks their answers
+    Then the user is not offered the chance to submit their appeal
```

Comment:
I was think that we could be even stronger here and say"... user cannot submit their appeal." This steps a bit away from UI because the underlying API will reject such a request anyway. In the context of the UI the user will not see the the submit button enabled so this will be the thing checked in the steps.